frontend/components/AddHostsModal/PlatformWrapper/_styles.scss

```diff
@@ -2,9 +2,14 @@
 
 .platform-wrapper {
   padding: 0; // different to pad sticky subnav properly
-  .react-tabs {
-    &__tab-list {
-      margin: 0 0 1.25rem;
+
```

Comment:
Is this expected to change in a "Prepare" PR?

Reply:
I think this may have been accidentally merged to main during the hard freeze period. @lukeheath want us to revert this commit here or you happy to keep it in the release branch?

cc @jacobshandling

Reply:
@rfairburn Thanks, that was not intended. 

@ghernandez345 It's okay to leave it in `main`, I cherry-picked it over to get it into the RC.